### PR TITLE
reorganize into sections

### PR DESCRIPTION
**Pull request by awokd**
Like this, @taradiddles ?

### CODE REVIEW

File: docs/misc/iaq.md [hunk 1]
```diff
@@ -1,10 +1,63 @@
...
+[Unit]
+ConditionPathExists=/var/run/qubes/this-is-netvm
+~~~
+
+### Xen passthrough compatible video cards
+
+- https://en.wikipedia.org/wiki/List_of_IOMMU-supporting_hardware#AMD
+- http://www.overclock.net/t/1307834/xen-vga-passthrough-compatible-graphics-adapters
+- https://wiki.xenproject.org/wiki/Xen_VGA_Passthrough_Tested_Adapters#ATI.2FAMD_display_adapters
+
```
Comment (taradiddles) [inline docs/misc/iaq.md]: Does this belong to "troubleshooting" ?
BTW that's the section I wanted to link to the hardware selection doc.

File: docs/misc/iaq.md [hunk 2]
```diff
@@ -40,22 +87,12 @@ See [this post](https://www.mail-archive.com/qubes-devel@googlegroups.com/msg028
 
 MaxFPS, UseDirtyBits.
 
+## Tweaks
+
 ### Where are VM log files kept?
 
 In the `/var/log/libvirst/libxl/`, `/var/log/qubes/` and `/var/log/xen/console/` directories.
```
Comment (taradiddles) [inline docs/misc/iaq.md]: I'd put this into the "troubleshooting" too

Comment (taradiddles) [reply docs/misc/iaq.md]: hmm ; I clicked on the '+' symbol on the left of "where are VM log..." but for some reason the "tweaks" stuff is green. Maybe that's how it ought to be.

File: docs/misc/iaq.md (hunk 1 above)
Comment (awokd) [reply docs/misc/iaq.md]: Which section should I go in? (And I know, but please do a PR against that doc.)

Comment (awokd) [reply docs/misc/iaq.md]: Not arguing, just not sure where else to put it. Maybe there should be another section, but I don't know what to call it! Merging for now.

Comment (taradiddles) [reply docs/misc/iaq.md]: don't worry ; I was also wondering where to put it - thought about a 'misc' section - but in the end 'troubleshooting' seemed OK.

**Pull request merged**